### PR TITLE
Split Position - Select Position

### LINKED ISSUE #75: [FE] - Split Position -  Select Position
- Columns to show - TBD
- Add button in Split Position
- Create new modal
- Table (similar to position list)
- Search and filter
- Select Position on click
- Submit

### PR DESCRIPTION
**Pull request by lmcorbalan**
closes #75

### CODE REVIEW

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr155--conditionaltokensexplorer.review.gnosisdev.com

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr155--conditionaltokensexplorer.review.gnosisdev.com

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr155--conditionaltokensexplorer.review.gnosisdev.com

File: src/pages/SplitPosition/SplitFrom.tsx [hunk 1]
```diff
@@ -76,7 +80,28 @@ export const SplitFrom: React.FC<Props> = (props) => {
...
   const [showCustomCollateralModal, setShowCustomCollateralModal] = useState(false)
+  const [showSelectPositionModal, setShowSelectPositionModal] = useState(false)
+
+  const openCustomCollateralModal = useCallback(() => setShowCustomCollateralModal(true), [])
+  const closeCustomCollateralModal = useCallback(() => setShowCustomCollateralModal(false), [])
+
+  const openSelectPositinModal = useCallback(() => setShowSelectPositionModal(true), [])
+  const closeSelectPositinModal = useCallback(() => setShowSelectPositionModal(false), [])
+
+  const onSelectPosidionModalConfirm = React.useCallback(
```
Comment (hpmaxi) [inline src/pages/SplitPosition/SplitFrom.tsx:95]: Typos

File: src/components/form/InputPosition/index.tsx [hunk 2]
```diff
@@ -1,93 +1,121 @@
...
-import { SplitPositionFormMethods } from '../../../pages/SplitPosition/Form'
-import { Error, ErrorContainer } from '../../pureStyledComponents/Error'
-import { Textfield } from '../../pureStyledComponents/Textfield'
-import { TitleControl } from '../../pureStyledComponents/TitleControl'
-import { TitleValue } from '../../text/TitleValue'
+import { GetMultiPositions_positions } from 'types/generatedGQL'
 
-const Span = styled.span``
+const canCanculatePositionToDisplay = (
+  positionsLoading: boolean,
```
Comment (hpmaxi) [inline src/components/form/InputPosition/index.tsx:16]: Typo

File: src/components/form/InputPosition/index.tsx [hunk 3]
```diff
@@ -1,93 +1,121 @@
...
+  balancesLoading: boolean,
+  positions: GetMultiPositions_positions[],
+  balances: BigNumber[],
+  positionIds: string[]
+) => {
+  return (
```
Comment (hpmaxi) [inline src/components/form/InputPosition/index.tsx:22]: This is the same as `canSetPositionBalance`?

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr155--conditionaltokensexplorer.review.gnosisdev.com

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr155--conditionaltokensexplorer.review.gnosisdev.com

File: src/components/form/InputAmount/index.tsx [hunk 4]
```diff
@@ -20,55 +24,83 @@ interface Props {
...
-          }
-        }
-      }
-
-      fetchBalance()
+    if (
+      splitFrom === 'position' &&
+      canSetPositionBalance(positionsLoading, balancesLoading, positions, balances, positionIds)
+    ) {
+      // FIXME - this only works with non custom tokens
```
Comment (mariano-aguero) [inline src/components/form/InputAmount/index.tsx:78]: Please use TODO, the linter throws a warning about it, or create an issue for this

**Review by mariano-aguero — COMMENTED** [review]
See comment

**Comment by mariano-aguero** [comment]
@lmcorbalan Something is broken regarding submit validation, see the picture, the button is disabled
Screenshot_20200902_164816

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr155--conditionaltokensexplorer.review.gnosisdev.com

**Comment by mariano-aguero** [comment]
Mmmm If I change from collateral to position tab/link, should my balance amount change from 0 to my collateral balance ?

Peek 02-09-2020 17-22

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr155--conditionaltokensexplorer.review.gnosisdev.com

**Review by mariano-aguero — APPROVED** [review]

**Pull request merged**